src/main/java/nextstep/courses/domain/SessionStudents.java

```diff
@@ -12,44 +12,74 @@
 import static java.util.stream.Collectors.toList;
 
 public class SessionStudents {
-    private Long sessionId;
-    private final List<Long> studentIds;
+    private List<SessionStudent> students;
+    private List<SessionStudent> approvedStudents = new ArrayList<>();
 
-    public SessionStudents(Long sessionId){
-        this(sessionId, new ArrayList<>());
+    public SessionStudents() {
+        students = new ArrayList<>();
     }
 
-    public SessionStudents(Long sessionId, List<Long> studentIds) {
-        this.sessionId = sessionId;
-        this.studentIds = studentIds;
+    public SessionStudents(List<SessionStudent> students) {
+        this.students = students;
     }
 
-    public void addStudent(Pricing pricing, NsUser user) {
-        checkAddStudent(pricing, user);
-        studentIds.add(user.getId());
+    public void addStudent(SessionStudent student) {
+        checkAddStudent(student);
+        students.add(student);
+    }
+
+    public void approveStudent() {
+        for (SessionStudent student : students) {
+            addApprovedStudent(student);
+        }
+    }
+
+    private void addApprovedStudent(SessionStudent student) {
+        if(student.isSelected()){
```

Comment:
IDE로 IntellIJ IDEA를 사용 중이시라면, 자동 포맷팅 단축키를 활용해보시는 걸 권장 드립니다.
(운영체제가 Windows라면 `Ctrl + Alt + L`, Mac이라면 `Cmd + Option + L`)
```suggestion
        if (student.isSelected()) {
```